wcivf/apps/elections/templates/elections/includes/_cancelled_election.html

```diff
@@ -8,6 +8,15 @@ <h2>Cancelled Election</h2>
     The poll for {{ object.post.label }} in the {{ object.election.name }}
     {% if object.election.in_past %}was{% else %}has been{% endif %}
     {% if object.replaced_by %}rescheduled{% else %}cancelled{% endif %}.
+
```

Comment:
I'm not quite sure about "Rescheduled Election" as a header - whether I can use the logic `{% if object.metadata.cancelled_election.title %}` or if not, the condition that I need to set.